### PR TITLE
FEATURE_entitlements_cou_nested_AARC_guideline

### PR DESCRIPTION
**Pull request by ioigoume**
Construct eduPersonEntitlements with VO hierarchy. For example:
```
urn:mace:foo.org:group:vo.parent.example.eu:vo.ioigoume.child.example.eu:role=member#foo.org
urn:mace:foo.org:group:vo.parent.example.eu:vo.child.example.eu:vo.child.child.example.eu:role=member#foo.org
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,8 @@ The following authproc filter configuration options are supported:
   * `urnAuthority`: Required, a string to use as the authority of the generated `eduPersonEntitlement` URN values containing group membership and role information.
   * `registryUrls`: Required, an array of COmanage endpoints representing standard Enrollment Flow types. All the four endpoints are mandatory.
   * `urnLegacy`: Optional, a boolean value for controlling whether to generate `eduPersonEntitlement` URN values using the legacy syntax. Defaults to `false`.
-  
+  * `mergeEntitlements`: A boolean to idicate whether the redundant `eduPersonEntitlement` will be removed from the state.
```
Comment (NicolasLiampotis) [inline README.md:71]: Default?

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]

**Comment by ioigoume** [comment]
Done

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**